osu.Game/Screens/Play/Player.cs

```diff
@@ -547,7 +547,7 @@ private IBeatmap loadPlayableBeatmap(Mod[] gameplayMods, CancellationToken cance
 
                 if (playable.HitObjects.Count == 0)
                 {
-                    Logger.Log("Beatmap contains no hit objects!", level: LogLevel.Error);
+                    Logger.Log("Beatmap contains no hit objects!", level: LogLevel.Important);
```

Comment:
Please change this back. This is an error condition.

Reply:
Why so? It's a handled exit that can be caused by users intentionally (not like an uncaught exception).
I can spot another condition in `Player` that looks like it should actually be shown as an error
https://github.com/ppy/osu/blob/668ce937a8d879ebf982ca4bdf1a8ccb1c6cfe36/osu.Game/Screens/Play/Player.cs#L202-L205

Reply:
> I can spot another condition in `Player` that looks like it should actually be shown as an error

sure, it probably should be. point is that it's an unexpected condition and it should be communicated to the user as such. i believe stable also posts an error-ish notification on this same circumstance.

Reply:
The game also posts `Important` level, FWIW. One may argue that important is more correct here because it's not the game erroring but the beatmap just being wrong, dunno.

https://github.com/ppy/osu/blob/cc800a18b203e474fc9fe6d43f50c2bc4e63ea03/osu.Game/OsuGame.cs#L1179-L1190

Reply:
> One may argue that important is more correct here because it's not the game erroring but the beatmap just being wrong

Yeah that's what makes more sense to me. 
Do Error level logs not also get reported to sentry? Seems very odd to report an issue not actually caused by the game itself as an error.

Reply:
> Don't Error level logs also get reported to sentry?

If there is no associated exception (which there is not in this case) then it will not be.

Reply:
> i believe stable also posts an error-ish notification on this same circumstance.

Stable handles this in an even worse way, just presenting an error of "beatmap could not be loaded"